generator/move_cross_mwm/move_cross_mwm.pro

```diff
@@ -0,0 +1,23 @@
+# Moves cross_mwm section from older mwm to new one, cross-referencing features with osm2ft.
```

Comment:
There is not cmake files for other projects in the generator directory, so I decided not to do one. I still use qmake for building.